docs/reference/firewall_requirements.md

```diff
@@ -0,0 +1,22 @@
+# Firewall requirements
+
+Firewall rules must be configured for Ubuntu Pro for WSL to operate fully.
+
+The following figure shows the possible connections between the different components and their default ports and protocols.
+
+![Firewall considerations.](./assets/firewall_requirements.png)
+
+The following table lists the default ports and protocols used by Ubuntu Pro for WSL:
+
+| Description | Client System | Server System | Protocol | Default Port | Target address |
+|-------------|---------------|---------------|----------|--------------|----------------|
+| Required for online installation of WSL instances.|Windows Host / Pro Agent|MS Store | tcp | 443 (https) | See [Microsoft documentation](https://learn.microsoft.com/en-us/microsoft-store/prerequisites-microsoft-store-for-business) for a list of addresses to allow. |
+| Ubuntu Pro enablement |Windows Host / Pro Agent |Canonical Contract Server |tcp |443 (https) | contracts.canonical.com |
+| Landscape management | Windows Host / Pro Agent | Landscape Server | tcp | 6554 (grpc) | On-premise Landscape address |
```

Comment:
```suggestion
| Landscape management | Windows Host / Pro Agent | Landscape Server | tcp | 6554 (gRPC) | On-premise Landscape address |
```

Reply:
I used small caps for all the protocols in the table.